### DISCUSSION TITLE
The doc: Installing Lammps may need update for 2.0.3

### DISCUSSION THREAD

**Discussion by likefallwind** [deepmodeling/deepmd-kit#1531] [Q&A] [answered]
Hi, I want to build lammps follow this way:https://docs.deepmodeling.org/projects/deepmd/en/master/install/install-lammps.html
![image](https://user-images.githubusercontent.com/10484087/156730430-4228ffae-bff7-4d2f-aaa2-99fe680d508a.png)

But I cannot find the dicretory: $deepmd_source_dir/source/build

Is there anything I do wrong? Or maybe we should update our doc?

**Comment by likefallwind** [comment]
This is deepmd_source_dir/source
![image](https://user-images.githubusercontent.com/10484087/156730741-3cec0bbc-af2e-4165-8d95-28c95ac5b8ed.png)

**Comment by wanghan-iapcm** [comment, marked as answer]
The `build` dir is made by the user https://docs.deepmodeling.org/projects/deepmd/en/master/install/install-from-source.html#install-the-deepmd-kit-s-c-interface

**Reply by likefallwind** [reply]
Thank you very much！
